### ISSUE TITLE
ATAG issues in Docassemble

### ISSUE THREAD

**Issue by BryceStevenWilley** [SuffolkLITLab/docassemble#15]
* Playground traps keyboard focus immediately, without a clear indication of how to exit or avoid ([A.3.1.2 No Keyboard traps](https://www.w3.org/TR/2015/NOTE-IMPLEMENTING-ATAG20-20150924/#sc_a312)
    * I believe this also goes for most instances of 
* Tab order on playground skips the Title and (more importantly) the main menu button (necessary for navigating back to other pages of docassemble)
* AAA, but A.3.2.4 Content Edits Saved (Extended) (i.e. autosave) should be added to playground edit screens, config screen, and (ideally) the fields during an interview.

(Still in progress, only got to A.3.2.4, will pickup again. https://www.w3.org/TR/ATAG20)
